src/subscription_manager/managercli.py

```diff
@@ -1687,11 +1687,11 @@ def _unbind_ids(self, unbind_method, consumer_uuid, ids):
 
     def _print_unbind_ids_result(self, success, failure, id_name):
         if success:
-            print _("%s successfully removed at the server:" % id_name)
+            print _("The entitlement server successfully removed these %s:" % id_name)
```

Comment:
The code changes here are fine, but I think the original code was incorrect.  I have some serious doubts as to how well these strings will display.  English is nice because you can just drop a noun into a sentence without worrying about it.  But other languages are going to have issues.  In Russian, for example, the `id_name` would be in the accusative case, but seeing the word "pool" in the `keys.pot` would probably cause a lot of translators to use the nominative case.  The user ends up seeing a grammatically incorrect sentence.

I think instead we should pass an entire sentence to `_print_unbind_ids_result`.